tests/vmi_multus_test.go

```diff
@@ -834,6 +834,22 @@ var _ = Describe("[Serial]SRIOV", func() {
 			// it's hard to match them.
 		})
 
+		It("should create a virtual machine instance without /sys/devices/ mounted on it's running pod", func() {
```

Comment:
do we really need this test ?
all the tests create VM, in case of such fundamental problem, we would see the error and i am not sure we want to verify on each and every time that the directory doesnt exists?

Reply:
well, I thought of it of way of proving the change actually does what we think it does. I wasn't sure it is needed either.
I can drop it if you prefer.

Reply:
IMO, we shouldnt check it, if we remove a mount, its not there, we should see that VM works and e2e pass thats all.
Maybe worth to mention the libvirt version where it was first not needed?

Reply:
Done.

> Maybe worth to mention the libvirt version where it was first not needed?

add to the commit message.